app/controllers/todos_controller.rb

```diff
@@ -0,0 +1,70 @@
+class TodosController < ApplicationController
+  before_action :set_todo, except: [:new, :index, :create]
+
+  def index
+    @complete = Todo.where(completed: true).sort_by &:duedate
+    @incomplete = Todo.where(completed: false).sort_by &:duedate

```

Comment:
Interesting and intuitive approach splitting the list this way.

Reply:
I didn't know we could sort by two parameters. I'll look in to that. That way sounds better.

Reply:
Todo.where(completed: true).order(:duedate)